docs/sources/visualizations/candlestick.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Candlestick"
+description = "Candlestick visualization documentation"
+keywords = ["grafana", "Candlestick", "OHLC", "panel", "documentation"]
+aliases = ["/docs/grafana/latest/features/panels/candlestick/", "/docs/grafana/latest/panels/visualizations/candlestick/"]
+weight = 600
++++
+
+# Candlestick
+
+The Candlestick panel allows you to visualize data which includes a number of consistent dimensions focused around price movement. The Candlestick panel also includes an Open-High-Low-Close (OHLC) mode, as well as support for additional dimensions based on time series data.
+
+{{< figure src="/static/img/docs/candlestick-panel/candlestick-panel-8-3.png" max-width="1200px" caption="Candlestick panel" >}}
+
+The Candlestick panel builds upon the foundation of the [time series]({{< relref "./time-series/_index.md" >}}) panel and includes many common configuration settings.
+
+## Mode
+
+The mode options allow you to toggle which dimensions are used for the visualization.
+
+- **Candles** limits the panel dimensions to the open, high, low and close dimensions used by candlestick visualizations.
+- **Volume** limits the panel dimension to the volume dimension.
+- **Both** is the default behavior for the candlestick panel. It includes both candlestick and volume visualizations.
+
+## Candle style
+
+- **Candles** is the default display style and creates candle-style visualizations between the open and close dimensions.
+- **OHLC Bars** displays the four core dimensions open, high, low and close values.
+
+## Color strategy
+
+- **Since Open** is the default behavior. This mode will utilize the _Up_ color (below) if the intra-period price movement is positive. In other words, if the value on close is greater or equal to the value on open, the _Up_ color is used.
+- **Since Prior Close** is an alternative display method based where the color of the candle is based the inter-period price movement or change in value. In other words, if the value on open is greater than the previous value on close, the _Up_ color is used. If the value on open is lower than the previous value on close, the _Down_ color is used. _This option also tiggers the hollow candlestick visualization mode_. Hollow candlesticks indicate that the intra-period movement is positive (value is higher on close than on open), filled candlesticks indicate the intra-period change is negative (value is lower on close than on open). [See here](https://thetradingbible.com/how-to-read-hollow-candlesticks) for a great explanation of the differences.
```

Comment:
```suggestion
- **Since Prior Close** is an alternative display method based where the color of the candle is based on the inter-period price movement or change in value. In other words, if the value on open is greater than the previous value on close, the _Up_ color is used. If the value on open is lower than the previous value on close, the _Down_ color is used. _This option also triggers the hollow candlestick visualization mode_. Hollow candlesticks indicate that the intra-period movement is positive (value is higher on close than on open), filled candlesticks indicate the intra-period change is negative (value is lower on close than on open). To learn more, see the [explanation of the differences](https://thetradingbible.com/how-to-read-hollow-candlesticks).
```